Aptoide/appcoins-wallet-android
# [bugfix/APPC-1426][bugfix/APPC-1427] - TopUp minimum limit bypass and backend request related fixes

**What does this PR do?**

   This PR fixes the issues described in the tickets APPC-1426 and APPC-1427, deals with entering in the TopUp screen with no network and deals with the possibility of not retrieving the chip values.

**Database changed?**

   No

**Where should the reviewer start?**

- TopUpFragmentPresenter.kt
- TopUpFragment.kt
- TopUpInteractor.kt
- TopUpValuesService.kt

**How should this be manually tested?**

Besides testing the issues described in the ticket, it should also be tested the TopUp flow without network.
  Flow on how to test this or QA Tickets related to this use-case: [APPC-1426][APPC-1427](https://aptoide.atlassian.net/browse/APPC-1426)
(https://aptoide.atlassian.net/browse/APPC-1427)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1426](https://aptoide.atlassian.net/browse/APPC-1426) [APPC-1427](https://aptoide.atlassian.net/browse/APPC-1427)


**Code Review Checklist**
